src/main.ts

```diff
@@ -22,7 +23,9 @@ async function main(): Promise<void> {
     const inputs = getActionInputs();
     await prepare();
     const {profileFolder} = await run(inputs);
-    await upload(inputs, profileFolder);
+    if (SKIP_UPLOAD !== "true") {
```

Comment:
I think we should handle this variable logic when we load it, so SKIP_UPLOAD would just be a boolean variable